File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int flag = 0;
        for(auto i:nums)
        {if(i==0) return 0;
        else if(i<0) flag++;}
        if(flag%2==0) return 1;
        else return -1;
    }
};

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        flag = 0
        for i in nums:
            if i==0:
                return 0
            elif i<0:
                flag -= 1
        if flag%2==0:
            return 1
        else:
            return -1